src/mcode.ts

```diff
@@ -596,13 +619,14 @@ export class ExtractedMCODE {
     // None of the conditions are satisfied.
     return 'NOT_SURE';
   }
-  getStageValue(): string {
+  getStageValues(): string[] {
+    var stageValues:string[] = [];
```

Comment:
This will be a lint error and should likely be `const` instead of `var`. (Confusingly you can edit a `const` array, you just can't assign a new array to that variable name.)

Reply:
Changed.